### ISSUE TITLE
please offer keyboard navigation in cinnamon-settings

### ISSUE THREAD

**Issue by VolMi** [linuxmint/Cinnamon#2523]
Please offer an intuitive way to navigate in cinnamon-settings using the keyboard.

I just effectively disabled my entire touchpad, by grubbing in the list of available panel applets and adding a faulty one. Currently lacking a mouse, I then tried to reach the mouse&touchpad module in cinnamon-settings but found no way to select it using the keyboard only.

Somehow, there actually _is_ some weird reaction to key presses in cinnamon-settings:
After startup, the search field has focus. If you press `tab`, the focus changes to one of the modules, but it seems impossible to stride through the available modules using arrow keys or `tab`. Instead, arrow key presses activate modules like you would expect it from pressing `Enter`.
